libraries/chain/include/eosio/chain/chain_config.hpp

```diff
@@ -281,14 +274,17 @@ template <typename DataStream>
 inline DataStream &operator<<(DataStream &s, const eosio::chain::data_entry<eosio::chain::chain_config_v1, eosio::chain::config_entry_validator> &entry){
    using namespace eosio::chain;
 
-   if (!entry.is_allowed())
-      return s;
+   //initial requirements were to skip packing field if it is not activated.
+   //this approach allows to spam this function with big buffer so changing this behavior
```

Comment:
Actually, skipping entries was wrong
- The contract has no way to know that the value was skipped and is likely to behave incorrectly.
- When the protocol feature is not activated, the old version of nodeos that doesn't know about the entry MUST behave the same as the new version of nodeos that does.  Skipping known but unactivated entries violates this.

Reply:
@swatanabe-b1  do you propose to remove the requirement for skipping/throwing exception in `get_parameters_packed` ? so inactive config should be presented regardless activation status

Reply:
I'm not suggesting any functional changes.  The comment indicates that this change is an optimization.  I'm saying that it's also necessary for correctness.  I think throwing an exception is the correct behavior for inactive config entries.